atlassian_jwt_auth/signer.py

```diff
@@ -9,11 +9,15 @@
 
 class JWTAuthSigner(object):
 
-    def __init__(self, issuer, private_key_retriever, **kwargs):
+    def __init__(self, issuer, private_key_retriever,
```

Comment:
This approach is commonly used by libraries to allow for ease of api change and not needing to keep adding arguments to the method signature.